dec-no-dos/dec_no_dos.cpp

```diff
@@ -0,0 +1,180 @@
+/** 	\file		dec_no_dos.cpp
+	\brief		Console intereface to run the program
+	\details	The aim of the program is to decrypt an EXE file encrypted with XOR with ECB mode. It is based on the fact that the key used in XOR encryption with ECB mode, will be easily spotable over a string of bytes of 0.
+	\author		Monika Olchowik
+	\date		12.2016
+	\version	1.0
+*/
+#include<getopt.h>
+#include<unistd.h>
+#include<cmath>
+#include<ctime>
+#include<cstdio>
+#include"dec.h"
+
+using namespace std;
+
+/**
+	\brief		Function printing the usage message of the function
+	
+	\arg		\c	argv	Arguments containing the name of the file
+*/
+void printf_usage(char** argv)
```

Comment:
I would say print_usage would be sufficient, since operation is called usually `print`, but later you might want to switch to `cout` with `std::format` for example.